tests/smoke/events.js

```diff
@@ -6,7 +6,10 @@
 (function () {
 	bender.editor = {
 		config: {
-			enterMode: CKEDITOR.ENTER_P
+			enterMode: CKEDITOR.ENTER_P,
+			nanospell: {
+				autostart: false
+			}
```

Comment:
autostart does horrible things to tests that rely on the spellcheck actually running, because depending on how fast your JS executes, it can finish spellchecking before your test actually has the listeners set up.  doh.